.github/workflows/test.yml

```diff
@@ -142,4 +142,4 @@ jobs:
         run: ./node_modules/.bin/devcontainer build --image-name vega-lite-devcontainer
 
       - name: Run yarn-build
-        run: docker run -ti -v `pwd`:/app -w /app vega-lite-devcontainer yarn build
+        run: docker run -ti -v `pwd`:/app -w /app vega-lite-devcontainer bash -c "sleep 1s && yarn build"
```

Comment:
This is unacceptable. Can't you just remote `-ti`?

Reply:
It's the opposite. "-it" activates the TTY.
But it's a known bug in Docker, that the activation of the TTY comes asynchronous and a bit too late.
The workaround is the "sleep".

Reply:
alternatively we can try:
-t 
-i
nothing

Reply:
I cannot trigger builds myself, can I ?

Reply:
You should be able to build in your branch as well.

Reply:
Btw, it worked without -it in my pull request.

Reply:
works now.